### PR TITLE
Feature/implement pagination 167683894

### PR DESCRIPTION
**Pull request by fisayomi**
#### What does this PR do?
- Implements pagination for products

#### Description of Task to be completed?
- A user should be able to set a page-limit of products to be displayed
- Update tests 

#### How should this be manually tested?
- Run the added queries in the tests with valid store and category as the case may be

### CODE REVIEW

File: komlie/utils/paginator.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from django.core.paginator import Paginator, InvalidPage
+
+
+def get_paginator(items, page_limit, page, paginated_type, **kwargs):
```
Comment (oreoluwade) [inline komlie/utils/paginator.py]: Good job ma'am. However, I must confess that the naming of this variable (when I saw where it was called before seeing the definition here) made me not know what its purpose was. Will it be possible to give it a name like maybe `paginate_items` or `get_paginated_items` or something similar? Also, could the variable `p` be more descriptive of what it does?

Comment (fisayomi) [reply komlie/utils/paginator.py]: Thank you for the feedback. Comment addressed!

**Comment by oreoluwade** [comment]
@fisayomi Could you please help us update the heroku travis key in the `.travis.yml` so our automatic builds can resume? The one in use at the moment is expired, hence, the builds fail when merged.

**Comment by fisayomi** [comment]
> @fisayomi Could you please help us update the heroku travis key in the `.travis.yml` so our automatic builds can resume? The one in use at the moment is expired, hence, the builds fail when merged.

Done @oreoluwade

File: komlie/utils/paginator.py (hunk 1 above)
Comment (oreoluwade) [reply komlie/utils/paginator.py]: Lol. Not completely addressed, but it's fine. Looks good to me. :)

Comment (fisayomi) [reply komlie/utils/paginator.py]: What is left to address @oreoluwade? I'm happy to work on it.

Comment (oreoluwade) [reply komlie/utils/paginator.py]: Nah. Nothing serious, just a difference in ideas around naming. Like I said, looks good to me.

Comment (fisayomi) [reply komlie/utils/paginator.py]: Okay, I used `items_to_paginate` cause it sounds more like what it is. `paginate_items` and `get_paginated_items` sounds like a function to me.

Comment (oreoluwade) [reply komlie/utils/paginator.py]: I was making reference to `get_paginator` itself, not the first argument of the `Paginator` constructor.

Comment (fisayomi) [reply komlie/utils/paginator.py]: Oh, I got it wrong then. Will fix in next PR.

**Pull request merged**